app/forms/UserForm.php

```diff
@@ -4,13 +4,18 @@
 use Phalcon\Forms\Element\Text;
 use Phalcon\Forms\Element\Password;
 use Phalcon\Forms\Element\Check;
+use Phalcon\Forms\Element\Hidden;
 use Phalcon\Validation\Validator\PresenceOf;
 use Phalcon\Validation\Validator\Email;
 use Phalcon\Validation\Validator\Confirmation;
 
 class UserForm extends Form{
 
-    public function initialize(){
+    public function initialize($user = null, $options = array()){
+
+        if($options['edit']){
+            $this->add(new Hidden("id"));
```

Comment:
Remove id from form. There is no need to pass the id since we can read it from url.